packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -235,11 +233,12 @@ Enumerates all Groups in your organization
 
 Reminders about the `limit` query parameter and query timeouts:
 
-* If you don't specify a value for `limit` and don't specify a query, only 200 results are returned for most orgs.
-* If you don't specify any value for `limit` and do specify a query, a maximum of 10 results are returned.
-* The maximum value for `limit` is 200 for most orgs.
-* Don't write code that depends on the default or maximum value, as it may change.
-* If you receive an HTTP 500 status code, you have more than likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
+- If you don't specify a value for `limit` and don't specify a query, only 200 results are returned for most orgs.
+- If you don't specify any value for `limit` and do specify a query, a maximum of 10 results are returned.
+- The maximum value for `limit` is 200 for most orgs.
+- Don't write code that depends on the default or maximum value, as it may change.
+- If you receive an HTTP 500 status code, you likely exceeded the request timeout. Retry your request with a smaller `limit` and [page the results](/docs/reference/core-okta-api/#pagination).
```

Comment:
@brianduffield-okta : Please reach out to UD team to spot check the accuracy and precision of above statements as it stands today. I know you didn't add them; while we are at it - @niravkumarpatel-okta fyi